### ISSUE TITLE
Vanishing / blowing up gradient during backpropagation

### ISSUE THREAD

**Issue by DanielEggelmann** [sagr4019/ResearchProject#84]
Research the meaning of vanishing or blowing up gradients during backpropagation and read the referenced paper "Learning long-term dependencies with gradient descent is difficult".

**Comment by DanielEggelmann** [comment]
Added wiki page:
https://github.com/sagr4019/ResearchProject/wiki/Vanishing---blowing-up-gradient

**Comment by DanielEggelmann** [comment]
Find or craft a concrete and simple example.

**Comment by DanielEggelmann** [comment]
Wiki article has been updated with example for gradients and gradient descent.
